src/ComponentPage.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+
+import './ComponentPage.scss'
+
+class ComponentPage extends Component {
+  renderSidebarLink(link, index) {
+    return (
+      <a
+        href={link.url}
+        key={index}
+        role='listitem'     
+        className={`mdc-list-item ${link.className} ${link.active ? 'sidebar-active' : ''}`}>
+          {link.content}
+       </a>
+    );
+  }
+
+  renderSidebar(activeLink) {
+    const links = [{
+      content: 'Buttons',
+      url: '/buttons.html',
+      className: 'sidebar-buttons',
+      active: activeLink === 'Buttons',
+    }, {
+      content: 'Cards',
+      url: '/card.html',
+      className: 'sidebar-cards',
+      active: activeLink === 'Cards',
+    }];
+
+    return(
+      <section className='sidebar mdc-layout-grid__cell mdc-layout-grid__cell--span-2'>
+        {links.map(this.renderSidebarLink)}
+      </section>
+    );
+  }
+
+  renderResource(title, imageSource, url) {
+    return (
+      <a href={url} role='listitem' className='mdc-list-item' target='_blank'>
+        <span className='mdc-list-item__graphic'><img src={imageSource} className='resources-icon'/></span>
+        <span className='mdc-list-item__text'>{title}</span>
+      </a>
+    );
+  }
+
+  renderDemoWrapper() {
+    return(
+      <section className='demo-wrapper mdc-layout-grid__cell mdc-layout-grid__cell--span-10'>
+        <h1 className='mdc-typography--headline'>{this.props.title}</h1>
+        <p className='mdc-typography--body1'>{this.props.description}</p>
+
+        <h2 className='mdc-typography--title'>Resources</h2>
+        {this.renderResource('Material Design Guidelines', 'material_design_black_24dp.png', this.props.designLink)}
+        {this.renderResource('Documentation', 'drive_document_black_24dp.png', this.props.docsLink)}
```

Comment:
Does it really make sense to include this link, when the source code link takes you to the package page which also shows the readme (which is the same as the documentation, except versioned)?

Reply:
I think it's important to have a link to MDC Web's section of material.io, to assert that the Material guidelines and MDC Web are related.